Formula/monax.rb

```diff
@@ -1,30 +0,0 @@
-class Monax < Formula
```

Comment:
Also remove symlink `Aliases/eris`

Reply:
OK, I did it while merging

Reply:
I should check notifications more often... 😞 Sorry for extra work @fxcoudert. Thank you.